docs/_dapp/native.md

```diff
@@ -3,64 +3,89 @@ title: Go API
 sort_key: C
 ---
 
-The Ethereum blockchain along with its two extension protocols Whisper and Swarm was
-originally conceptualized to become the supporting pillar of web3, providing the
-consensus, messaging and storage backbone for a new generation of distributed (actually,
-decentralized) applications called DApps.
+Ethereum was originally conceptualized to be the base layer for [Web3][web3-link], providing 
+the backbone for a new generation of decentralized, permissionless and censorship resistant 
+applications called [dapps][dapp-link]. The first step towards this vision was the development 
+of clients providing an RPC interface into the peer-to-peer protocols. This allowed users to 
+transact between accounts and interact with smart contracts using command line tools. 
+Geth was one of the original clients to provide this type of gateway to the Ethereum network.
 
-The first incarnation towards this dream of web3 was a command line client providing an
-RPC interface into the peer-to-peer protocols. The client was soon enough extended with a
-web-browser-like graphical user interface, permitting developers to write DApps based on
-the tried and proven HTML/CSS/JS technologies.
+Before long, web-browser-like graphical interfaces (e.g. Mist) were created to extend clients.
+However, as dapps became more complex their requirements exceeded what a simple browser environment
+could support. To support the next generation of dapps, built using time-tested HTML/CSS/JS 
```

Comment:
Heh, this isn't your sentence, but it's IMO a pretty bad sentence even from the beginning. "what a simple browser environment could support" -- a modern browser environment can support some pretty darn complex things.
 
I'm not sure how to better phrase it , but programmatic access to a larger/deeper API is at the core, I think.

Reply:
ok I agree it is clunky. Will try to reword.